client/webpack.config.js

```diff
@@ -0,0 +1,27 @@
+var webpack = require('webpack');
```

Comment:
maybe const/let for the variables in this file to stay consistent with the other client files